src/lib/stores/linearTeams.ts

```diff
@@ -0,0 +1,72 @@
+import { browser } from '$app/environment';
+import { writable } from 'svelte/store';
+import type { Team } from '../linear';
+import { toast } from '@significa/svelte-ui';
+
+const getDefaultTeams = () => {
+  if (browser) {
+    const localStoreTeams = window.localStorage.getItem('linear-teams');
+
+    if (localStoreTeams) {
+      try {
+        JSON.parse(localStoreTeams);
+      } catch (e) {
+        throw new Error();
+      }
+
+      const parsedTeams = JSON.parse(localStoreTeams);
+
+      if (Array.isArray(parsedTeams)) {
+        return parsedTeams;
+      } else {
+        throw new Error();
+      }
+    }
+  }
+  return [];
+};
```

Comment:
This is quite confusing for me.
I recommend removing all the `throws` because you don't want to throw any error, in case of error you should return a default value which is an empty array. Remember you're populating the store with the return of this function.
So, basically, I would wrap the whole function in a `try ... catch` the catch is a return default array.

```suggestion
const getDefaultTeams = () => {
  try {
    if (browser) {
      const localStoreTeams = window.localStorage.getItem('linear-teams');
  
      if (localStoreTeams) {
        const parsedTeams = JSON.parse(localStoreTeams);
  
        if (Array.isArray(parsedTeams)) {
          return parsedTeams;
        }
      }
    }
  } catch(err) {
    console.error(err);
  }
  // will return default state
  return [];
};
```